packages/@tinacms/toolkit/src/packages/react-sidebar/components/Sidebar.tsx

```diff
@@ -85,11 +87,10 @@ const useFetchCollections = (cms) => {
           const response = await cms.api.admin.fetchCollections()
           setCollections(response.getCollections)
         } catch (error) {
-          cms.alerts.error(
-            `[ERROR] GetCollections failed: ${error.message}`,
-            30 * 1000 // 30 seconds
-          )
           setCollections([])
+          throw new Error(
+            `[${error.name}] GetCollections failed: ${error.message}`
+          )
```

Comment:
I'm not so sure about these changes (this one and https://github.com/tinacms/tinacms/pull/2763/files#diff-8a0c2ffb8744070329e971b8234f082257bdba440d694187711aa7d779436abf) as they related to the GraphQL backend failing to respond correctly.  I believe these were written to fail more... gracefully than resulting in the ErrorBoundary?

If we _do_ want to handle CMS errors in the same way, there's a few more in the `/admin` folder:
* packages/tinacms/src/admin/pages/CollectionCreatePage.tsx, line 152
* packages/tinacms/src/admin/pages/CollectionUpdatePage.tsx, line 135